### ISSUE TITLE
Sound volume on 1.5.1_RC2

### ISSUE THREAD

**Issue by cyberpunkrocker-zero** [dhewm/dhewm3#338]
I just upgraded dhewm3 from1.5.0 to 1.5.1_RC2 on my Arch Linux system. While the 1.5.0 was quite quiet already, on 1.5.1_RC2 the sound volume was so low as being nearly inaudible. Because I have a dual-head system, I opened pavucontrol on the secondary screen and started dhewm3. There I noticed that the playback level on dhewm3 audio stream was for some reason set to 60% by default (And I definitely have not set it myself!!!). But, because dhewm3 grabs the mouse, I couldn't re-set the volume normally; I had to SSH in from my laptop and set the volume via a ncurses pulsemixer to 100%. 

But still, dhewm3 maximum sound volume is very low, much more silent than on my other games, for example. I had to set the system volume to 120-130% before launching dhewm3 to set the volume about the same level as in other games. I remember the original Doom3 being much louder... (different machine, though). The in-game audio slider is set to max, and there's no difference in volume whether eax is on or not.

**Comment by DanielGibson** [comment]
If you open the console (Shift+Esc) the mouse gets ungrabbed.

In my experience pulseaudios per-application volume is a broken mess; I'm not sure if that's specific to the pulse plugin for the XFCE panel (which also handles keyboard shortcuts for setting the volume) or a general problem, but on my system changing the global volume (either with the volume controls of my keyboard or by using the mousewheel on the volume icon on the panel) changes both the default output devices volume *and* the volume of all active playback sources. It also seems  to remember the configured volume when restarting applications.

If it's the same on your system that could explain why dhewm3 has a different volume in pulse than other applications.

**Comment by DanielGibson** [comment]
Oh, also, yes, dhewm3 is less loud than Doom3 used to be, to work around a sound issues with lots of loud sounds (like shooting the shotgun against a nearby metal wall), see #179

**Comment by cyberpunkrocker-zero** [comment]
No, I only use pavucontrol app to adjust volume, it does not behave like you described. And it still does not change the fact that dhewm3 has way too low maximum audio volume.
I should probably mention that I use Awesome wm (no DE's) and output the audio via GTX1070 HDMI/DisplayPort to my display's loudspeakers.

**Comment by cyberpunkrocker-zero** [comment]
(sigh) Well damn, maybe I should make a script which raises the volume before launching dhewm3, and resets it back when dhewm3 closes...

**Comment by DanielGibson** [comment]
Then I have no idea where the 60% are coming from, dhewm3 shouldn't change system volume settings.

You can also set `s_volume_dB` in the console to values bigger than 0 (try 5 or 10) to make the game louder.
(When setting the slider in the settings menu to max it's 0, this allows you to go higher)

**Comment by James90000** [comment]
Hello, I have been trying to fix the issue outlines in:

https://github.com/dhewm/dhewm3/issues/179

For a silly amount of time, I don't know how many hours.

I have tried reducing the volume with s_volume_dB, I have tried downloading and configuring OpenAL, I have tried disabling EAX 4.0 in game.

Essentially if a shotgun is used in close quarters, you can't really hear it fire at all and instead just heard 20 pellets hitting a metal wall.

I have no issue in the vanilla game, but I would really like to use your port if I can get the sound working properly.

Thank you,

James

**Comment by James90000** [comment]
Update:

I reduced the 3 audio files that play when bullets hit metal by 18db in Audacity and the loaded them after the base game files.

Drawback is that for non-shotguns, the sound of bullets hitting the wall is near inaudible,
The upside is that the sound of bullets hitting the wall is awful and I doubt most people would know that it is even missing.

I am beyond relieved.

**Comment by DanielGibson** [comment]
Is this still an issue?
Over the years I made several attempts to improve the sound volume issues and nowadays there are two settings related to it in the dhewm3 Settings menu under Audio Options:
`[ ] Scale down and clamp all sound volumes`
And `OpenAL output limiter` which can be explicitly enabled/disabled.

Furthermore, newer OpenAL Soft versions could also help.

**Comment by James90000** [comment]
Hi Daniel,

At the time I couldn’t fix this issue after many attempts.

So I made this mod where I took the impacts sounds from Doom 3 and lowered their volumes 
by about 18db. 

This fixes the issue, and I personally prefer it with no impact sounds, the gun makes enough sound on its own so it’s not noticeable that those sounds are essentially missing.

I haven’t played Doom 3 (or Dhewm 3) since summer last year, so I’m not sure if this is still an ongoing issue.

If you like I can download Dhewm 3 and check it out?

**Comment by DanielGibson** [comment]
Well, if you have a solution that you're happy with then I'm happy.
Would be interesting if dhewm3 1.5.4 with its new OpenAL options make a difference for you, but you it's not that important.

And I guess in the end it's hard to avoid that 20 pellets hitting a metal wall at close distance are very loud. But at least in my tests after some tweaking the shotgun itself could be heard at all, which was an improvement.

**Comment by James90000** [comment]
I just checked and I'm afraid the issue is still there.
